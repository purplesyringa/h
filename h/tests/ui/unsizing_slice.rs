fn main() {
    let unsizing_slice = h::map! {
        &[1, 2, 3][..] => (),
        &b"abc"[..] => (),
    };
}
