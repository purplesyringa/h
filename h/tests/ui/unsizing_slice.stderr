error: unsupported expression
       `h` is not a full-blown interpreter; only certain operators are allowed
 --> tests/ui/unsizing_slice.rs:3:20
  |
3 |         &[1, 2, 3][..] => (),
  |                    ^^

error: indexing is not supported
       `h` is not a full-blown interpreter
       to unsize an array, use `as &[_]`
 --> tests/ui/unsizing_slice.rs:3:10
  |
3 |         &[1, 2, 3][..] => (),
  |          ^^^^^^^^^^^^^

error: unsupported expression
       `h` is not a full-blown interpreter; only certain operators are allowed
 --> tests/ui/unsizing_slice.rs:4:17
  |
4 |         &b"abc"[..] => (),
  |                 ^^

error: indexing is not supported
       `h` is not a full-blown interpreter
       to unsize an array, use `as &[_]`
 --> tests/ui/unsizing_slice.rs:4:10
  |
4 |         &b"abc"[..] => (),
  |          ^^^^^^^^^^
